content/en/guides/core/registry/model_registry/access_controls.md

```diff
@@ -21,7 +21,7 @@ For example, suppose you set `staging` and `production` as protected aliases. An
 ## Set up access control
 The following steps describe how to set up access controls for your team’s model registry.
 
-1. Navigate to the W&B Model Registry app at [https://wandb.ai/registry/model](https://wandb.ai/registry/model).
+1. Navigate to the [W&B Model Registry app](https://wandb.ai/registry/model).
```

Comment:
Interesting, I thought it was best practice to show the URL for product-specific links?

Reply:
I've never heard that rule of thumb, but Google says this: https://developers.google.com/style/cross-references#descriptive-link-text

A screen reader can easily tell the reader what the URL is and it's super annoying to be forced to encounter the URL when you are just reading the words and you suddenly have to deal with a pretty ugly sentence . A sighted reader can hover over the link to see the URL. WDYT?